devfiles/php-laravel/devfile.yaml

```diff
@@ -0,0 +1,99 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: php-laravel
+projects:
+-
+  name: laravel-realworld-example-app
+  source:
+    type: git
+    location: "https://github.com/gothinkster/laravel-realworld-example-app.git"
+components:
+-
+  type: chePlugin
+  id: redhat/php/latest
+  memoryLimit: 1000Mi
+-
+  type: chePlugin
+  id: redhat/php-debugger/latest
+  memoryLimit: 256Mi
+-
+  type: dockerimage
+  alias: tools
```

Comment:
is there something except php inside? Can it be named as `php`?

Reply:
there is nothing special, changed to `php`